### ISSUE TITLE
spectaql 2.0.1 installation error

### ISSUE THREAD

**Issue by robinelvin** [anvilco/spectaql#605]
I'm trying to install for the first time:

```shell
yarn add -D spectacl

...prior output omitted for brevity

➤ YN0000: │ spectaql@npm:2.0.1 STDOUT
➤ YN0000: │ spectaql@npm:2.0.1 STDOUT > spectaql@2.0.1 build:vendor
➤ YN0000: │ spectaql@npm:2.0.1 STDOUT > node bin/build-vendor.mjs
➤ YN0000: │ spectaql@npm:2.0.1 STDOUT
➤ YN0000: │ spectaql@npm:2.0.1 STDERR No vendor-src directory. Not building vendor packages.
➤ YN0009: │ spectaql@npm:2.0.1 couldn't be built successfully (exit code 13, logs can be found here: /private/var/folders/p6/s92jvcdj1m37h71ygtl8cs_r0000gn/T/xfs-02057cca/build.log)
➤ YN0000: └ Completed in 5s 800ms
➤ YN0000: Failed with errors in 14s 863ms
```

Using Yarn 3.3.0 PnP on MacOS

While the package appears to be installed this error means my CI build fails.

EDIT: Whilst this fails on initial installation it only raises a warning when using `yarn install --immutable`

**Comment by newhouse** [comment]
Weird. Seems to be related to this: https://github.com/nodejs/node/issues/40808

**Comment by newhouse** [comment]
Looks like you are using Yarn 3.3.0, but what version of Node are you on @robinelvin ?

**Comment by newhouse** [comment]
Can you clear things out and try things again with `2.0.2-beta.0` and let me know if that resolves the issue for you?

**Comment by robinelvin** [comment]
> Looks like you are using Yarn 3.3.0, but what version of Node are you on @robinelvin ?

Good catch.

```
node -v
v16.13.0
```

> Can you clear things out and try things again with `2.0.2-beta.0` and let me know if that resolves the issue for you?

```shell
➤ YN0002: │ spectaql@npm:2.0.2-beta.0 doesn't provide jquery (p59002), requested by foundation-sites
➤ YN0002: │ spectaql@npm:2.0.2-beta.0 doesn't provide motion-ui (p2f3f8), requested by foundation-sites
➤ YN0002: │ spectaql@npm:2.0.2-beta.0 doesn't provide what-input (p8395b), requested by foundation-sites
➤ YN0000: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code
➤ YN0000: └ Completed in 1s 876ms
➤ YN0000: ┌ Fetch step
➤ YN0013: │ spectaql@npm:2.0.2-beta.0 can't be found in the cache and will be fetched from the remote registry
➤ YN0019: │ spectaql-npm-2.0.1-466251f8f5-1023c13732.zip appears to be unused - removing
➤ YN0000: └ Completed in 1s 337ms
➤ YN0000: ┌ Link step
➤ YN0008: │ @elixirsoftware/core@workspace:. must be rebuilt because its dependency tree changed
➤ YN0007: │ spectaql@npm:2.0.2-beta.0 must be built because it never has been before or the last one failed
➤ YN0000: │ @elixirsoftware/core@workspace:. STDOUT husky - Git hooks installed
➤ YN0000: │ spectaql@npm:2.0.2-beta.0 STDOUT
➤ YN0000: │ spectaql@npm:2.0.2-beta.0 STDOUT > spectaql@2.0.2-beta.0 build:vendor
➤ YN0000: │ spectaql@npm:2.0.2-beta.0 STDOUT > node bin/build-vendor.mjs
➤ YN0000: │ spectaql@npm:2.0.2-beta.0 STDOUT
➤ YN0000: │ spectaql@npm:2.0.2-beta.0 STDERR No vendor-src directory. Not building vendor packages.
➤ YN0000: └ Completed in 2s 489ms
➤ YN0000: Done with warnings in 6s 82ms
```

Seems better, thanks.

**Comment by robinelvin** [comment]
I've uninstalled and reinstalled and it appears to be happy now. Calling `process.exit(0)` looks like it fixes the problem.
Thanks

**Comment by newhouse** [comment]
Fix is out in `2.0.2`. Thanks/sorry!
